dev-build/app/AppLoader.scala

```diff
@@ -24,7 +24,6 @@ import services.ophan.SurgingContentAgentLifecycle
 import play.api.ApplicationLoader.Context
 import play.api._
 import play.api.routing.Router
-import play.filters.csrf.{CSRFAddToken, CSRFCheck}
```

Comment:
These have been  annoying me for so long. 

I think they're a hangover from email sign up work and we should make a note (maybe) to look into these unused CSRF tokens across the platform.

Reply:
+1